core/embed/rust/src/translations/blob.rs

```diff
@@ -258,6 +258,19 @@ impl<'a> Translations<'a> {
     pub fn header<'b>(&'b self) -> &'b TranslationsHeader<'b> {
         &self.header
     }
+
+    /// Returns a pointer to the glyph data for the given UTF-8 codepoint.
```

Comment:
update the comment please

Reply:
5ae0e784a26b8f4a1baa37816bf083baff6f603b